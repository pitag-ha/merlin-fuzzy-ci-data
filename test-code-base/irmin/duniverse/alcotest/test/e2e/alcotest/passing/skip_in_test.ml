let () =
  let open Alcotest in
  let id () = () in
  run __FILE__
    [
      ( "test-a",
        [
          test_case "Run test case" `Quick id;
          test_case "Skip test case" `Quick skip;
        ] );
    ]
