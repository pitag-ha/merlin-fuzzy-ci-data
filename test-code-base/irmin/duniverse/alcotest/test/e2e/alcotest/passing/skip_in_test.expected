Testing `test/e2e/alcotest/passing/skip_in_test.ml'.
This run has ID `<uuid>'.

  [OK]          test-a          0   Run test case.
  [SKIP]        test-a          1   Skip test case.

Full test results in `<build-context>/_build/_tests/<test-dir>'.
Test Successful in <test-duration>s. 1 test run.
